src/renderer_mtl.mm

```diff
@@ -3181,6 +3181,10 @@ void writeString(bx::WriterI* _writer, const char* _str)
 		{
 			m_metalLayer.displaySyncEnabled = 0 != (_flags&BGFX_RESET_VSYNC);
 		}
+		if (@available(macOS 10.13.2, *))
+		{
+			m_metalLayer.maximumDrawableCount = (_flags&BGFX_RESET_VSYNC) ? 2 : 3;
```

Comment:
You should respect `numBackBuffers` from `bgfx::Resolution` here.

Reply:
Ah didn't realize there was a setting for that.
I changed it around to use that and am clamping to keep within the accepted range.

Setting numBackBuffers to 3 will still create this bug though. 2 is the default though so should fix it for most people?